src/Common/perf/PerfRunner/PerfRunner.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Reflection;
+using System.Collections.Generic;
+using Microsoft.Xunit.Performance.Api;
+
+public class PerfHarness
+{
+    public static void Main(string[] args)
+    {
+        using (XunitPerformanceHarness harness = new XunitPerformanceHarness(args))
```

Comment:
Perfect.  Thanks.  May be worth breaking that one up at some point, but not a requirement for this change.